nse_main.lua

```diff
@@ -662,6 +662,45 @@ do
   Script.__index = Script;
 end
 
+local function get_script_name(tbl)
+  local o={}
+  for _, val in pairs(tbl) do
+    if type(val) == "string" then
+      table.insert(o,val)
+    end
+  end
+  return o
+end
+
+local function make_parser()
+  local script_file_name = C( (1-S'/=%:|\"<>.,')^1 )
+  local without_argument = P","^0
```

Comment:
Few unnecessary naming conventions can be changed here if we are not planning to committing the script-arguments, as soon as we are done with this PR.